hummingbot/client/command/gateway_command.py

```diff
@@ -92,12 +84,12 @@ def test_connection(self):
         safe_ensure_future(self._test_connection(), loop=self.ev_loop)
 
     def gateway_config(self,
-                       key: List[str],
+                       key: str = None,
```

Comment:
Since the parameter can be `None` the type hint should be Optional: `key: Optional[str] = None,`

Reply:
Hi, this has been changed!